system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -651,10 +661,15 @@ mod benches {
 		}
 
 		fn set_up_complex_asset_transfer() -> Option<(Assets, u32, Location, Box<dyn FnOnce()>)> {
-			// BH only supports teleports to system parachain.
-			// Relay/native token can be teleported between BH and Relay.
+			// Only supports native token teleports to system parachain
 			let native_location = Parent.into();
-			let dest = Parent.into();
+			let dest = AssetHubLocation::get();
+
+			// TODO: Remove below line once we update to polkadot-sdk stable2503
```

Comment:
Why is that? Could you maybe add the issue or PR relevant for this?

Reply:
We will fix TODOs as a part of https://github.com/polkadot-fellows/runtimes/issues/696

And iirc, this PR: https://github.com/paritytech/polkadot-sdk/pull/6604/files#diff-299b81a18ba51e2c2fdaa3b6c7c7dd99ced49c9c7b8772fa1225ebecced332e0R237-R243 is not in the stable2412, but it is in stable2503